### PR TITLE
Feature/allergen card

### PR DESCRIPTION
**Pull request by SaschaRadojkovic**
added a components folder with a AllergeneCard folder with an index.js which is maping over an array of objects to render a list of allergens
issue [#1](https://github.com/SaschaRadojkovic/capstone-project/issues/1)

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import AllergenCard from "@/components/AllergeneCard";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      <AllergenCard></AllergenCard>
```
Comment (ThomasF85) [inline pages/index.js:6]: You can write your component as self closing tag:
```
<AllergenCard />
```

File: components/AllergeneCard/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    id: 19,
+    name: "lactose",
+  },
+  {
+    id: 20,
+    name: "vanillin",
+  },
+];
+
+export default function AllergenCard() {
```
Comment (ThomasF85) [inline components/AllergeneCard/index.js:84]: The name of the component is misleading. It is actually a list of items or cards.

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
Looks good except for two minor requested changes.

File: components/AllergeneCard/index.js (hunk 2 above)
Comment (SaschaRadojkovic) [reply components/AllergeneCard/index.js:84]: ok .should i name it AllergenList ? because it is no card ?

**Review by ThomasF85 — APPROVED** [review]
Looks good!

**Pull request merged**